### ISSUE TITLE
Bug: Site Check notification is sent when everything is ok

### ISSUE THREAD

**Issue by boldgrid-support** [BoldGrid/boldgrid-backup#89]
**Describe the bug**
Some users are getting notifications every time Site Check completes, even when there is no follow up action needed.

**To Reproduce**
1. Configure a notification email address.
2. Set Site Check to run every minute
3. Receive email every minute that everything is OK

**Expected behavior**
Notifications should only go out when errors are detected.

**Example Message**
```Content-Type: message/rfc822
Content-Transfer-Encoding: 7bit
Content-Disposition: attachment

Received: from emaildomain.com by emaildomain.com with LMTP
 id GNEZKm+ICF2B6QYAOJBE0w
 (envelope-from <user@emaildomain.com>)
 for <user@emaildomain.com>; Tue, 18 Jun 2019 00:45:03 -0600
Received: from user by emaildomain.com with local (Exim 4.92)
 (envelope-from <user@emaildomain.com>) id 1hd7rb-001tqQ-Kr
 for user@emaildomain.com; Tue, 18 Jun 2019 00:45:03 -0600
Return-Path: <user@emaildomain.com>
From: "\(Cron Daemon\)" <user@emaildomain.com>
To: <user@emaildomain.com>
Subject: Cron <user@emaildomain> php -d register_argc_argv="1" -qf
 "/home4/user/public_html/wp-content/plugins/boldgrid-backup/cli/bgbkup-cli.php"
 check auto_recovery=0 email=info
Date: Tue, 18 Jun 2019 07:45:03 +0100
Message-ID: <E1hd7rb-001tqQ-Kr@emaildomain.com>
MIME-Version: 1.0
Content-Type: text/plain;
	charset="UTF-8"
Content-Transfer-Encoding: 7bit
X-Mailer: Microsoft Outlook 14.0
Thread-Index: AQIPTKJmqo/b812VJGaZfZbZReu/Zg==

Site Check: Ok
Info: No action taken.

**Comment by gidders28** [comment]
What is happening with this? The last comment was 9 days ago

I'm getting 48 emails / day that just say
Site Check: Ok
Info: No action taken.

I'm getting fed up with my inbox being bombarded

Regards, Clive

**Comment by bwmarkle** [comment]
Hi @gidders28,

I'm really sorry that you're getting bombarded with these false positive emails. I'm looking into this, and will get back with you later today.

Thanks,
- Brad

**Comment by bwmarkle** [comment]
Hi @gidders28,

Is this the full subject of the email?

```
Subject: Cron <user@emaildomain> php -d register_argc_argv="1" -qf
 "/home4/user/public_html/wp-content/plugins/boldgrid-backup/cli/bgbkup-cli.php"
 check auto_recovery=0 email=info
```

It appears that `>/dev/null 2>&1` is missing from the cron command, and I'm not sure if you omitted that or if the system did?

Based on that being the subject, it appears that the server's cron system is emailing you, and not the BoldGrid Backup plugin. But either way, BoldGrid Backup is related and I think we can help get this solved, we just need a bit more info about the exact cron job command being ran (IE the subject of the email).

Thanks!
- Brad

**Comment by gidders28** [comment]
Hi Brad

That's the email subject as it appears - I've not omitted anything. Also if it helps, the emails are addressed to clivegid@box5757.bluehost.com

Strange thing is that I only get these emails on my phone - they don't register in Outlook on my desktop PC

However "Backup completed for clivegidneyphotography.com" confirmations come through on both phone & desktop addressed to  info@clivegidneyphotography.com

Regards, Clive

**Comment by bwmarkle** [comment]
Hi Clive,

I think we have a hunch of what the issue is. Before we implement a temporary fix for testing, to confirm, you have still been receiving these emails in the last hour? We resaved your cron settings, and I want to make sure that us resaving the settings did not fix the issue.

Thanks,
- Brad

**Comment by gidders28** [comment]
Still receiving them every 15 minutes - last received 16.45

Clive

**Comment by cssjoe** [comment]
We found an issue with the Linux system parsing the cron entry; everything after the percent sign (%) is truncated.  We will be pushing out an update shortly.

**Comment by gidders28** [comment]
Shortly as in a few hours? Or a few days?

**Comment by bwmarkle** [comment]
Hi Clive, we'll have a release hopefully within the next hour or two. We'll keep you updated.

**Comment by cssjoe** [comment]
We just released version 1.10.2 containing the fix for the cron entry issue causing cron systems to send output to the default/configured email address.  Please let us know if you do have any further issues.  We will be happy to help you.
